### DISCUSSION TITLE
0.1.6

### DISCUSSION THREAD

**Discussion by SSujitX** [SSujitX/google-news-url-decoder#11] [Announcements]
### 0.1.6 (8th October 2024)

- Version 0.1.6:
  - Improved: Enhanced error handling with a fallback mechanism for decoding parameters.
  - Refined: Optimized get_decoding_params to try decoding via https://news.google.com/articles first, falling back to https://news.google.com/rss/articles if needed
  - Updated: Reduced occurrences of HTTP 429 (Too Many Requests).
  - Removed: Logging #9  functionality for a cleaner codebase.
  - Fixed: Resolved time delay issue between requests.

**Full Changelog**: https://github.com/SSujitX/google-news-url-decoder/compare/0.1.5...0.1.6

<hr /><em>This discussion was created from the release <a href='https://github.com/SSujitX/google-news-url-decoder/releases/tag/0.1.6'>0.1.6</a>.</em>

**Comment by mt-sizilio** [comment]
@SSujitX This lib is wonderful!!! 🥇 
I discovered it recently and it helped me a lot, but unfortunately it stopped working recently. I can't get the URLs anymore.

**Reply by SSujitX** [reply]
Can you show, what is the issue?
